2016-31-08-dont-panic

Mrida opened the pull request.

bil-t commented:
the build is failing

On Thu, Sep 1, 2016 at 8:52 AM, Mrida notifications@github.com wrote:

> ---
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/murex/murex-coding-dojo/pull/21
> Commit Summary
> - 2016-31-08-dont-panic
> 
> File Changes
> - _A_ Beirut/2016/2016-31-09-dont-panic/Main.java
>   https://github.com/murex/murex-coding-dojo/pull/21/files#diff-0 (63)
> - _A_ Beirut/2016/2016-31-09-dont-panic/readme.md
>   https://github.com/murex/murex-coding-dojo/pull/21/files#diff-1 (69)
> 
> Patch Links:
> - https://github.com/murex/murex-coding-dojo/pull/21.patch
> - https://github.com/murex/murex-coding-dojo/pull/21.diff
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/murex/murex-coding-dojo/pull/21, or mute the thread
> https://github.com/notifications/unsubscribe-auth/AKZMWC_usKZSHUl7BiAVXa9uX8sYR8Dbks5qlnYegaJpZM4JyXuA
> .

aatwi commented:
I was't aware there was a build for this! I think we need to add a rake file to the job in order to make it pass!

Mrida commented:
I have added a .travis.yml file with java language set and the correct maven script. I also added a pom.xml file. Seems that Travis is not able to see my .travis.yml file.
